tests/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import requests
+from urllib.parse import urljoin
+from urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+TEST_PASSWORD = "fakef00bar"
+
+pytest_plugins = ["docker_compose"]
+
+
+@pytest.fixture(scope="session")
+def argus_api(wait_for_argus_api):
+    """Returns the URL and a valid token for a running Argus API server"""
+
+    request_session, container = wait_for_argus_api
+    container.execute(["django-admin", "initial_setup"])
+    container.execute(["django-admin", "setpassword", "admin", TEST_PASSWORD])
+
+    service = container.network_info[0]
+    argus_url = f"http://127.0.0.1:{service.host_port}/"
+
+    response = request_session.post(
+        urljoin(argus_url, "/api/v1/token-auth/"),
+        data={"username": "admin", "password": TEST_PASSWORD},
+    )
+    token = response.json().get("token", None)
+    assert token, f"Failed to get token from {response.payload}"
+
+    return "http://localhost:8000/api/v2", token
```

Comment:
Needs a fixture for /api/v2/token/login/ as well. I guess the auth-bit can be broken out into its own function.

Reply:
I couldn't get `/api/v2/token/login/` to work when I tried it, so using the old endpoint was faster than figuring out why not.